tests/inventory.test.js

```diff
@@ -161,6 +161,50 @@ describe("Given Inventory is connected to NATS", () => {
                     });
                 });
             });
+
+            describe("And when I transfer the item", () => {
```

Comment:
tests look good. 👍 

Remember to run `npm run prettier` on your code before creating the PR, there are some formatting issues here